common/constants/shared.ts

```diff
@@ -61,13 +61,13 @@ export const observabilityMetricsTitle = 'Metrics';
 export const observabilityMetricsPluginOrder = 5092;
 
 export const observabilityTracesNewNavID = 'observability-traces-nav';
-export const observabilityTracesNewNavURL = observabilityTracesNewNavID + '#/traces';
+export const observabilityTracesNewNavURL = observabilityTracesNewNavID;
 export const observabilityTracesID = 'observability-traces';
 export const observabilityTracesTitle = 'Traces';
 export const observabilityTracesPluginOrder = 5093;
 
 export const observabilityServicesNewNavID = 'observability-services-nav';
-export const observabilityServicesNewNavURL = observabilityServicesNewNavID + '#/services';
+export const observabilityServicesNewNavURL = observabilityServicesNewNavID;
```

Comment:
If we are assigning the `observabilityTracesNewNavURL = observabilityTracesNewNavID;` Can we just use the observabilityTracesNewNavID directly?

Reply:
nit. Not sure why we have `ID` capitalized should be `Id` for at all places.

Reply:
Looks like all ID's are in this file Ill do a separate PR to fix as it will affect a lot of files and don't want the other changes to get overshadowed by it.